### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SnowLew**
Updated readme to easly navigation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,154 +46,149 @@ Semantic Versioning is used, and corresponds to electron versions in the followi
...
+sudo apt-get install xvfb # or equivalent
+export DISPLAY=':99.0'
+Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
+electron-pdf ...
 ```
 
-There is also an example docker machine [here](https://github.com/fraserxu/docker-tape-run).
+A Docker machine example is available [here](https://github.com/fraserxu/docker-tape-run).
+
+Please see [the npm docs](https://docs.npmjs.com/misc/config#unsafe-perm) for more information.
```
Comment (jerryorr) [inline README.md:76]: This probably made more sense where it was, up where `--unsafe-perm` is used

File: README.md [hunk 2]
```diff
@@ -22,154 +46,149 @@ Semantic Versioning is used, and corresponds to electron versions in the followi
...
-### You can use the same instance
+#### Using Electron-PDF
 
 ```javascript
-var ElectronPDF = require('electron-pdf')
-var express = require('express')
-var bodyParser = require('body-parser')
-var app = express()
-app.use(bodyParser.json())
+const ElectronPDF = require('electron-pdf');
```
Comment (jerryorr) [inline README.md:97]: Can we undo all these semi-colon and tabs vs spaces changes? Not interested in a code-style war 😆

**Review by jerryorr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SnowLew) [reply README.md]: Done 1/2

File: README.md (hunk 2 above)
Comment (SnowLew) [reply README.md]: Done 2/2

**Pull request merged**